sips/sip-prioritized-transaction-submission.md

```diff
@@ -0,0 +1,79 @@
+| SIP-Number          |  |
+| ---:                | :--- |
+| Title               | Prioritized Transaction Submission |
+| Description         | Enable effective prioritized transaction submission. |
+| Author              | Shio Coder <shio.coder@gmail.com> |
+| Editor              |  |
+| Type                | Standard |
+| Category            | Core |
+| Created             | 2024-11-26 |
+| Comments-URI        |  |
+| Status              |  |
+| Requires            | N/A |
+
+
+## Abstract
+
+Enable effective prioritized transaction submission based on gas price.
+
+
+## Motivation
+
+Currently, there is no deterministic gas ordering for the transactions, even when these transactions are submitted within an extremely short time window (less than 100ms). This is due to the fact that different transactions are randomly submitted by different validators, and the validator that is supposed to submit the transaction can have latency jitters. We propose a mechanism to mitigate the random factor and enable prioritized transaction submission.
```

Comment:
I think this is not widely known, and making gas prices work more like folks would intuitively expect is valuable.